orchestra/bots/staffbot.py

```diff
@@ -214,3 +217,20 @@ def _send_staffing_request_by_slack(self, staffing_request, message):
         self.slack.post_message(worker.slack_user_id, message)
         staffing_request.status = StaffingRequest.Status.SENT.value
         staffing_request.save()
+
+    def validate(self, data):
+        """
+        Handle a request received from slack. First we validate the
+        request and then pass the message to the appropriate handler.
+        """
+        slack_user_id = data.get('user_id')
+        username = data.get('user_name')
+
+        worker = get_object_or_None(Worker, slack_user_id=slack_user_id)
+        if worker is None:
+            raise SlackUserUnauthorized('Worker {} not found'.format(username))
```

Comment:
We should return a json response instead of raising. Otherwise the end user will just get the HTML for a 500 page back in slack